plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/PostgreConstants.java

```diff
@@ -153,6 +153,9 @@
         DATA_TYPE_ALIASES.put("time without time zone", TYPE_TIME);
         DATA_TYPE_ALIASES.put("timestamp with time zone", TYPE_TIMESTAMPTZ);
         DATA_TYPE_ALIASES.put("timestamp without time zone", TYPE_TIMESTAMP);
+        
+        DATA_TYPE_ALIASES.put("character", TYPE_BPCHAR);
```

Comment:
https://github.com/dbeaver/dbeaver/blob/9679ae7766b65aa0544527a4f2712707890e79e3/plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/impls/redshift/PostgreServerRedshift.java#L269

I think, we already have it here.

Reply:
Yes, but that's not visible and the BPCHAR and VARCHAR are applicable to both Postgres and Redshift.

Reply:
@LonwoLonwo Does this change look good or was there any specific reason to keep varchar and bpchar in PosgreServerRedshift class alone?

Reply:
@serge-rider Any inputs on my question above. Should I do something different in the code?

Reply:
@LonwoLonwo Did you get a chance to look at my question? Was there any specific reason to keep varchar and bpchar in PosgreServerRedshift class alone?

Reply:
@deepaksenthilkumar Actually, I already added these aliases some time ago, so you don't have to:
https://github.com/dbeaver/dbeaver/blob/ab82296b92eaf6ea7a135e4da017617078b26179/plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/PostgreConstants.java#L148-L151

Reply:
@ShadelessFox-Cool. Then I will do one thing... Rebase the code based on current devel branch and then just change 
[RedshiftView.java](plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/impls/redshift/)

Reply:
@ShadelessFox - I have completed that. Now the changes are just in 

[RedshiftView](plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/impls/redshift/RedshiftView.java)
And
[PostgreServerRedshift](plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/impls/redshift/PostgreServerRedshift.java)

Reply:
👍🏻